src/status_im2/subs/wallet/networks.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im2.subs.wallet.networks
+  (:require [quo.foundations.resources :as resources]
+            [re-frame.core :as re-frame]
+            [status-im2.constants :as constants]))
+
+(re-frame/reg-sub
+ :wallet/filtered-networks-by-mode
+ :<- [:wallet/networks]
+ (fn [networks [_ test?]]
+   (get networks (if test? :test :prod))))
+
+(def ^:const mainnet-network-details
+  {:source       (resources/networks :ethereum)
+   :short-name   "eth"
+   :network-name :ethereum})
+
+(def ^:const arbitrum-network-details
+  {:source       (resources/networks :arbitrum)
+   :short-name   "arb1"
+   :network-name :arbitrum})
+
+(def ^:const optimism-network-details
+  {:source       (resources/networks :optimism)
+   :short-name   "opt"
+   :network-name :optimism})
+
+(def ^:const network-list
+  {constants/mainnet-chain-id  mainnet-network-details
+   constants/arbitrum-chain-id arbitrum-network-details
+   constants/optimism-chain-id optimism-network-details})
+
+(re-frame/reg-sub
+ :wallet/network-details
```

Comment:
A quick test capturing the outputs when `is-test` is true or false would be good.

Reply:
oh yeah this I can test. Was planning to do this once the integration tests were ready. I created that issue but we have someone else working on it now fyi :)